### ISSUE TITLE
Please merge the .mk files

### ISSUE THREAD

**Issue by DevUt** [Hax-on/android_device_zte_ailsa_ii#2]
Spiting the makefiles has made debugging a tedious task
Please merge all the flags in the board dir into the BoardConfig

**Comment by Celoxocis** [comment]
Yes i agree! Who ever at CM came up with that idea should be executed the Blood Eagle way...
I merged them in my device tree here: https://github.com/Celoxocis/android_device_zte_ailsa_ii_new
I haven't updated my device tree in a while since BMC joined the Dev effort.

**Comment by msfjarvis** [comment]
Having named fragments is more manageable IMO
